sound/soc/sof/intel/hda.h

```diff
@@ -153,7 +153,9 @@
 /* FW and ROM share offset 4 */
 #define HDA_DSP_SRAM_REG_FW_STATUS		(HDA_DSP_MBOX_OFFSET + 0x4)
 #define HDA_DSP_SRAM_REG_FW_TRACEP		(HDA_DSP_MBOX_OFFSET + 0x8)
-#define HDA_DSP_SRAM_REG_FW_END			(HDA_DSP_MBOX_OFFSET + 0xc)
+#define HDA_DSP_SRAM_REG_FW_IPC_RECEIVED_COUNT	(HDA_DSP_MBOX_OFFSET + 0xc)
+#define HDA_DSP_SRAM_REG_FW_IPC_PROCESSED_COUNT	(HDA_DSP_MBOX_OFFSET + 0x10)
+#define HDA_DSP_SRAM_REG_FW_END			(HDA_DSP_MBOX_OFFSET + 0x14)
```

Comment:
@plbossart FW_END is just like `end` values popular in enums, it marks that there are no more registers. As I see it's not used by the driver, but I updated it, because it already was there and equivalent of it in FW code `SRAM_REG_FW_END` is actually used on FW side.

Reply:
ok thanks